docs/references/files/fossa-yml.md

```diff
@@ -61,10 +61,10 @@ Sets the [FOSSA API key](https://docs.fossa.com/docs/api-reference#api-tokens) t
 ### `project:`
 The project fields allow you to configure settings for the project you are interacting with through the FOSSA API.
 
-> Note: `name`, `team`, `policy`, `link`, `url`, and `jiraProjectKey` can only be set when creating a project (running `fossa analyze` for the first time).
+> Note: `name`, `team`, `policy`, `link`, and `jiraProjectKey` can only be set when creating a project (running `fossa analyze` for the first time).
```

Comment:
I'd like to add a little more here, though I know this wasn't the intention of the change:

```suggestion
> Note: `name`, `team`, `policy`, `link`, and `jiraProjectKey` can only be set when creating a project (running `fossa analyze` for the first time).  Otherwise, they will be silently ignored (we would like to make this a visible warning in the future).
```